### PR TITLE
Added my profile in LinkFree/public/data

### PR DESCRIPTION
**Pull request by RajPakhurde**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1206"></a>

### CODE REVIEW

File: public/data/RajPakhurde.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/RajPakhurde",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/raj_pakhurde",
+        "icon": "twitter"
+      }
+      {
```
Comment (nhcarrigan) [inline public/data/RajPakhurde.json:17]:
```suggestion
      },
      {
```
Need a comma between array entries.

File: public/data/RajPakhurde.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started Front-end web-development",
+        "date": "December 2021",
+        "icon": "dollar",
+        "color": "grey",
+        "description": "Started learning web-development",
+        "url": "rajpakhurde.showwcase.com"
```
Comment (kumarsonsoff3) [inline public/data/RajPakhurde.json:30]: Fix this, because this may sometimes take you to an unwanted error! (404)

```suggestion
        "url": "https://rajpakhurde.showwcase.com"
```

File: public/data/RajPakhurde.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      }
+    ]
+  }
```
Comment (nhcarrigan) [inline public/data/RajPakhurde.json:33]:
```suggestion
  }

```
Empty new-line at the end of the file.

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
Heya!

A couple of fixes for the linter. 🙂

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: public/data/RajPakhurde.json (hunk 2 above)
Comment (RajPakhurde) [reply public/data/RajPakhurde.json:30]: Ok i got it i will update that link.

**Comment by RajPakhurde** [comment]
Now please check i have updated that url

File: public/data/RajPakhurde.json (hunk 1 above)
Comment (RajPakhurde) [reply public/data/RajPakhurde.json]: Now i have added comma and removed empty line at end

File: public/data/RajPakhurde.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/raj_pakhurde",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on Linkdein",
+        "url": "https://www.linkedin.com/in/raj-pakhurde-78b297228/",
```
Comment (vikasganiga05) [inline public/data/RajPakhurde.json:19]:
```suggestion
        "url": "https://linkedin.com/in/raj-pakhurde-78b297228/",
```
Remove `www.` from the url.

File: public/data/RajPakhurde.json [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started Front-end web-development",
+        "date": "December 2021",
+        "icon": "dollar",
+        "color": "grey",
+        "description": "Started learning web-development",
+        "url": "https://www.showwcase.com/rajpakhurde"
```
Comment (vikasganiga05) [inline public/data/RajPakhurde.json:30]:
```suggestion
        "url": "https://showwcase.com/rajpakhurde"
```
Remove `www.` from the url.

**Review by vikasganiga05 — CHANGES_REQUESTED** [review]
It's my suggestion to remove `www.` from the URL. It does not make any changes. Other URLs not include `www.` in them. So, following the same convention looks good.

File: public/data/RajPakhurde.json (hunk 5 above)
Comment (RajPakhurde) [reply public/data/RajPakhurde.json:30]: ok i will remove it.

File: public/data/RajPakhurde.json (hunk 3 above)
Comment (Pradumnasaraf) [reply public/data/RajPakhurde.json:33]: Hey! Naomi, I guess the extra line addition will be handled by the workflow

**Review by Pradumnasaraf — APPROVED** [review]
LGTM!, thank you for your contribution :rocket: 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Efficient, Maintainable

File: public/data/RajPakhurde.json (hunk 5 above)
Comment (Panquesito7) [reply public/data/RajPakhurde.json]: Note that some sites do not work without the `www.`, so be careful with that. 🙂

**Comment by RajPakhurde** [comment]
Thanks Pradumna!

**Review by schmelto — APPROVED** [review]
## Looking good to me

### TLDR;

Thank you for the PR. Your suggested changes looking good to me.


---

### Full details

Thank you for the PR. Your suggested changes looking good to me.
Feel free to checkout other issues. 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by krshdev — APPROVED** [review]
Looks great to me!
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

File: public/data/RajPakhurde.json [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/raj_pakhurde",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on Linkdein",
+        "url": "https://linkedin.com/in/rajpakhurde",
+        "icon": "linkdein"
```
Comment (Shivansh-yadav13) [inline public/data/RajPakhurde.json:20]:
```suggestion
        "icon": "linkedin"
```
this should fix the icon

File: public/data/RajPakhurde.json [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+        "url": "https://github.com/RajPakhurde",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/raj_pakhurde",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on Linkdein",
```
Comment (Shivansh-yadav13) [inline public/data/RajPakhurde.json:18]:
```suggestion
        "name": "Follow me on LinkedIn",
```

File: public/data/RajPakhurde.json [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/raj_pakhurde",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://linkedin.com/in/rajpakhurde",
```
Comment (kabir0x23) [inline public/data/RajPakhurde.json:19]:
```suggestion
        "url": "https://www.linkedin.com/in/raj-pakhurde-78b297228",
```

**Review by kabir0x23 — CHANGES_REQUESTED** [review]
Your LinkedIn URL Link is broken

**Review by krshdev — APPROVED** [review]

**Comment by krshdev** [comment]
Oh sorry, didn't see the LinkedIn link broken review

**Review by Pradumnasaraf — APPROVED** [review]
LGTM

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Maintainable

**Review by emmalearnscode — CHANGES_REQUESTED** [review]
Everything looks good apart from the broken linkedIn link. If you can update it to a working one we can get this merged in 👍 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by kabir0x23 — APPROVED** [review]
Amazing, Looks good to me, 🚀

File: public/data/RajPakhurde.json [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+      {
+        "title": "Started Front-end web-development",
+        "date": "December 2021",
+        "icon": "dollar",
+        "color": "grey",
+        "description": "Started learning web-development",
+        "url": "https://showwcase.com/rajpakhurde"
+      }
+    ]
+  }
```
Comment (Panquesito7) [inline public/data/RajPakhurde.json:33]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
